modules/tinymce/src/themes/silver/test/ts/browser/editor/NotificationManagerImplTest.ts

```diff
@@ -114,21 +114,21 @@ describe('browser.tinymce.themes.silver.editor.NotificationManagerImplTest', ()
       assertStructure('Check success notification structure', nSuccess, 'success', 'Message 4');
 
       // Check items are positioned so that they are stacked
-      assertPosition('Error notification', nError, 220, -200);
-      assertPosition('Warning notification', nWarn, 220, -152);
-      assertPosition('Info notification', nInfo, 220, -104);
-      assertPosition('Success notification', nSuccess, 220, -56);
+      assertPosition('Error notification', nError, 220, -400);
```

Comment:
I wonder if instead of these tests relying on the default height if we should specify it in the init configuration so it won't change (like we do with the width)?

Reply:
I set the height to 200 and reverted other changes. It looks cleaner.